vllm/distributed/kv_transfer/kv_connector/v1/nixl_connector.py

```diff
@@ -1542,6 +1664,20 @@ def get_finished(self) -> tuple[set[str], set[str]]:
                 self.sync_recved_kv_to_device(req_id, meta)
             if self.enable_permute_local_kv:
                 block_ids_to_permute += meta.local_block_ids
+
+            # post processing for heteroblocksize
+            block_size_ratio = self.kv_topo.block_size_ratio_from_engine_id(
+                meta.remote_engine_id
+            )
+            if (
+                not self.use_mla
+                and block_size_ratio != 1
```

Comment:
nit: `block_size_ratio > 1` due to original remote < local constraint. It just helps keeping things clearer and consistent

Reply:
done